cpp/third_party/vcpkg_overlays/triplets/x64-linux-mix.cmake

```diff
@@ -0,0 +1,9 @@
+set(VCPKG_TARGET_ARCHITECTURE x64)
+set(VCPKG_CRT_LINKAGE dynamic)
+set(VCPKG_CMAKE_SYSTEM_NAME Linux)
+
+if(${PORT} MATCHES "openssl")
+	set(VCPKG_LIBRARY_LINKAGE dynamic)
```

Comment:
I'm wondering shouldn't we require dynamic linking to the standard library as well in this case?

Reply:
I think so but maybe not in the scope of this PR?